sdk/evaluation/azure-ai-evaluation/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,39 @@
+# Troubleshoot AI Evaluation SDK Issues
+
+This guide walks you through how to investigate failures, common errors in the `azure-ai-evaluation` SDK, and steps to mitigate these issues.
+
+## Table of Contents
+
+- [Handle Evaluate API Errors](#handle-evaluate-api-errors)
+  - [Troubleshoot Remote Tracking Issues](#troubleshoot-remote-tracking-issues)
+  - [Safety Metric Supported Regions](#safety-metric-supported-regions)
+- [Handle Simulation Errors](#handle-simulation-errors)
+  - [Adversarial Simulation Supported Regions](#adversarial-simulation-supported-regions)
+- [Logging](#logging)
+- [Get additional help](#get-additional-help)
+
+## Handle Evaluate API Errors
+
+### Troubleshoot Remote Tracking Issues
+
+- If your AI Studio hub is set to identity-based storage access and you're unable to upload data because of permission issues, check whether the `Storage Blob Data Contributor` role is assigned to the storage account linked to your Azure AI Studio hub. You can find more details [here](https://review.learn.microsoft.com/azure/ai-studio/how-to/disable-local-auth).
+
+- Additionally, if you're using a virtual network or private link, and your evaluation run upload fails because of that, check out this [guide](https://docs.microsoft.com/azure/machine-learning/how-to-enable-studio-virtual-network#access-data-using-the-studio).
+
```

Comment:
Please add: 

Before running `evaluate()`, to ensure that you can enable logging and tracing to your Azure AI project, make sure you are first logged in by running `az login`.

Then install the following sub-package:

```python
pip install azure-ai-evaluation[remote]
```

Finally, ensure that you assign the proper storage permissions for CosmosDB. This can be done with the following command:

```Shell
az role assignment create --role "Storage Blob Data Contributor" --scope /subscriptions/<mySubscriptionID>/resourceGroups/<myResourceGroupName> --assignee-principal-type User --assignee-object-id "<user-id>"
```

Reply:
Updated